.env

```diff
@@ -0,0 +1 @@
+PYTHONPATH=lib:src
```

Comment:
Do we really need this file?

Reply:
This file is not needed in prod, and is only for VSCode to recognize the lib directory. I'll remove this from the upstream source and make git ignore this file.